src/logic/calculate.js

```diff
@@ -4,6 +4,15 @@ function isNumber(item) {
   return !!item.match(/[0-9]+/);
 }
 
+/**
+ * Given a button name and a calculator data object, return an updated
+ * calculator data object.
+ *
+ * Calculator data object contains:
+ *   total:s      the running total
+ *   next:String       the next number to be operated on with the total
+ *   operation:String  +, -, etc.
+ */
```

Comment:
- [ ] [OPTIONAL] Kindly remove the unused lines of code in order for us to have a clean code-base.